acapy_agent/did/cheqd/routes.py

```diff
@@ -53,6 +123,125 @@ class CreateResponseSchema(OpenAPISchema):
     )
 
 
+class DeactivateRequestSchema(OpenAPISchema):
+    """Parameters and validators for deactivate DID endpoint."""
+
+    did = fields.Str(
+        required=True,
+        validate=CHEQD_DID_VALIDATE,
+        metadata={"description": "DID to deactivate", "example": CHEQD_DID_EXAMPLE},
+    )
+    options = fields.Dict(
+        required=False,
+        metadata={
+            "description": "Additional configuration options",
+            "example": {"network": "testnet"},
+        },
+    )
+
+
+class DeactivateResponseSchema(OpenAPISchema):
+    """Response schema for deactivate DID endpoint."""
+
+    did = fields.Str(
+        validate=CHEQD_DID_VALIDATE,
+        metadata={
+            "description": "DID that has been deactivted",
+            "example": CHEQD_DID_EXAMPLE,
+        },
+    )
+    did_document = fields.Dict(
+        required=False,
+        allow_none=True,
+        metadata={
+            "description": "The DID document, if available, after deactivation. \
+            For deactivated DIDs, this is usually set to None.",
+        },
+    )
+    did_document_metadata = fields.Dict(
+        required=True,
+        metadata={
+            "description": "Metadata related specific to the DID document, \
+            indicating status changes. This typically includes a 'deactivated' status \
+            flag to confirm the operation.",
+        },
+    )
+
+
+class UpdateRequestSchema(OpenAPISchema):
+    """Parameters and validators for update DID endpoint."""
+
+    EXAMPLE = {
+        "did": CHEQD_DID_EXAMPLE,
+        "didDocumentOperation": "setDidDocument",
+        "didDocument": {
+            "id": CHEQD_DID_EXAMPLE,
+            "controller": [CHEQD_DID_EXAMPLE],
+            "verificationMethod": [
+                {
+                    "id": CHEQD_DID_EXAMPLE + "#key-1",
+                    "type": "Ed25519VerificationKey2018",
+                    "controller": CHEQD_DID_EXAMPLE,
+                    "publicKeyMultibase": "z6Mk...",
+                }
+            ],
+            "authentication": [CHEQD_DID_EXAMPLE + "#key-1"],
+            "service": [
+                {
+                    "id": CHEQD_DID_EXAMPLE + "#service-1",
+                    "type": "MessagingService",
+                    "serviceEndpoint": ["https://example.com/service"],
+                }
+            ],
+        },
+        "options": {"network": "testnet"},
+    }
+
+    did = fields.Str(
+        required=True,
+        validate=CHEQD_DID_VALIDATE,
+        metadata={"description": "DID to update"},
+    )
+    options = fields.Dict(
+        required=False,
+        metadata={
+            "description": "Additional configuration options",
+            "example": {"network": "testnet"},
+        },
+    )
+    didDocumentOperation = fields.Str(
+        required=True,
+        validate=validate.OneOf(
+            ["setDidDocument", "removeFromDidDocument", "addToDidDocument"]
```

Comment:
Core cheqd registrar only supports "setDidDocument" though

Reply:
Removed 'didDocumentOperation' and used 'setDidDocument' as the only option in Manager